python/ray/train/v2/_internal/state/state_actor.py

```diff
@@ -20,18 +25,54 @@ def __init__(self):
         # {run_id: {attempt_id: TrainRunAttempt}}
         self._run_attempts: Dict[str, Dict[str, TrainRunAttempt]] = defaultdict(dict)
 
+        self._export_logger = self._init_export_logger()
+
+    def _init_export_logger(self) -> Optional[logging.Logger]:
+        # Proto schemas should be imported within the scope of TrainStateActor to
+        # prevent serialization errors.
+        from ray.core.generated.export_event_pb2 import ExportEvent
+
+        if ray_constants.RAY_ENABLE_EXPORT_API_WRITE:
```

Comment:
I think this needs to check both this env var and the `RAY_ENABLE_EXPORT_API_WRITE_CONFIG_STR` env var. I think there is a python helper function that does this.